src/Modules/RoomsModule/components/RoomsData/RoomsData.tsx

```diff
@@ -1,7 +1,283 @@
+              </Alert>
+            )}
+          </Grid>
+          <Grid item md={6}>
+            <TextField
+              label="Capacity"
+              variant="outlined"
+              {...register("capacity", {
+                required: "capacity is required",
+              })}
+              fullWidth
+            />
+            {errors.capacity && (
+              <Alert severity="error" sx={{ mt: 1 }}>
+                {errors.capacity.message}
+              </Alert>
+            )}
+          </Grid>
+          <Grid item md={6}>
+            <TextField
+              label="Discount"
+              variant="outlined"
+              {...register("discount", {
+                required: "discount is required",
+              })}
+              fullWidth
+            />
+            {errors.discount && (
+              <Alert severity="error" sx={{ mt: 1 }}>
+                {errors.discount.message}
+              </Alert>
+            )}
+          </Grid>
+          <Grid item md={6}>
+            <FormControl fullWidth>
+              <InputLabel>Facilities</InputLabel>
+              <Select
+                multiple
+                value={watch("facilities") || []}
+                {...register("facilities", {
+                  required: "facilities is required",
+                })}
+                sx={{ width: "100%" }}
+                renderValue={(selected) => (
+                  <div>
+                    {selected.map((id: string) => {
+                      const facility = facilitiesList.find(
+                        (facility: FacilitiesProps) => facility._id === id
+                      );
+                      return (
+                        <span style={{ margin: "8px" }} key={id}>
+                          {facility ? facility?.name : "Facility not found"}
+                        </span>
+                      );
+                    })}
+                  </div>
+                )}
+              >
+                {facilitiesList.map((item: FacilitiesProps) => (
+                  <MenuItem key={item._id} value={item?._id}>
+                    <Checkbox
+                      checked={watch("facilities")?.includes(item._id)}
+                    />
+
+                    <ListItemText primary={item?.name} />
+                  </MenuItem>
+                ))}
+              </Select>
+            </FormControl>
+            {errors.facilities && (
+              <Alert severity="error" sx={{ mt: 1 }}>
+                {errors.facilities.message}
+              </Alert>
+            )}
+          </Grid>
+          <Grid item md={12}>
+            <label htmlFor="imgs">
+              <Button
+                component="span"
+                role={undefined}
+                variant="contained"
+                tabIndex={-1}
+                startIcon={<CloudUploadIcon />}
+              >
+                Upload Images
+              </Button>
+            </label>
+
+            <input
```

Comment:
I suggest having a containr and make the input take its full width and height and make its opacity 0. and make the styling we want and make it fill the container as well.
The input input should be on the top. This will allow us to drag and drop the images without selecting them. As input file is a droppable area by default.

Reply:
Fixed RoomsData and added container